.nvmrc

```diff
@@ -0,0 +1 @@
+v6.17.1
```

Comment:
Could this be updated to a later version of Node? I think v10 is the latest that Slate v0 worked on, and if we're adding other dependencies to it, it's probably safest to do so with the latest version of Node we can.